src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -277,6 +288,15 @@ inline void FUNC(fc_bf_tiled_kernel_default)(
     ACCUMULATOR_TYPE* d_zps = (ACCUMULATOR_TYPE*)(&d_zp);
 #endif
 
+#if OUTER_OFM > 1
+    unroll_for (uint oi = 0; oi < OUTER_OFM; ++oi) {
+#if OUTPUT_3D
+    input_offset = out_b0 * INPUT0_BATCH_PITCH + out_b1 * INPUT0_FEATURE_PITCH + INPUT0_OFFSET;
+#else
+    input_offset = out_b * TILE_IN_B_PITCH + INPUT0_OFFSET;
```

Comment:
How about just store the input_offset_init and reuse it instead of recalculating it?

Reply:
applied